<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1452</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_11">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabFenPrinc">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGraph">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Graphique</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="1" rowspan="2" colspan="2">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="1">
               <widget class="QPushButton" name="buttonSupprGroupe">
                <property name="text">
                 <string>Supprimer Groupe</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="buttonAjoutGroupe">
                <property name="text">
                 <string>Ajouter Groupe</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" rowspan="2">
               <widget class="QTreeView" name="treeViewGroupe">
                <property name="minimumSize">
                 <size>
                  <width>500</width>
                  <height>150</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QPushButton" name="buttonZoomP">
                <property name="maximumSize">
                 <size>
                  <width>85</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Zoom +</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="buttonZoomM">
                <property name="maximumSize">
                 <size>
                  <width>85</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Zoom -</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>199</width>
              <height>39</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="buttonLegende">
            <property name="text">
             <string>Légende</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>42</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1" rowspan="2">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="labelTabStations">
              <property name="text">
               <string>Stations</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QListView" name="listViewStations">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>350</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>700</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="2" rowspan="2">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_12">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QFrame" name="frameGraph">
              <property name="minimumSize">
               <size>
                <width>800</width>
                <height>400</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="0" column="0">
                <widget class="QGraphicsView" name="graphView">
                 <property name="styleSheet">
                  <string notr="true">border-color: rgb(0, 0, 0);</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="3" rowspan="2">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="labelTabParametre">
              <property name="text">
               <string>Paramètres</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QListView" name="listViewParametre">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>350</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>700</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-color: rgb(0, 0, 0);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="4">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>42</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="2">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QLabel" name="labelDate">
                <property name="text">
                 <string>Date</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDateTimeEdit" name="dateTimeDate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="displayFormat">
                 <string>yyyy-mm-dd hh:mm:ss</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="labelTaux">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Taux</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditTaux">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="5" column="2">
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>891</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabValidation">
       <attribute name="title">
        <string>Validation</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1452</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionFermer"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
